examples/tests/managed-ng-os/index.ts

```diff
@@ -54,6 +54,16 @@ const managedNodeGroupAL2023 = eks.createManagedNodeGroup("al-2023-mng", {
   nodeRole: role,
 });
 
+// Create a node group with a launch template and custom AMI
+const managedNodeGroup = new eks.ManagedNodeGroup("al-2023-mng-amitype-launch-template", {
+  ...scalingConfig,
+  cluster: cluster,
+  nodeRole: role,
+  amiType: "AL2023_ARM_64_STANDARD",
```

Comment:
Should this instead warn the user to not specify amiType here since it is a no-op, or it actually takes effect in other parts of the system?

Reply:
It actually takes effect. This causes the ami id to be set in the launch template